### ISSUE TITLE
document possible requirement for libgl1-mesa-glx

### ISSUE THREAD

**Issue by ksbeattie** [CCSI-Toolset/FOQUS#515]
When running foqus.py on my Ubuntu VMs, I have to `apt-get install libgl1-mesa-glx` otherwise I get an 
```
2019-06-04 22:41:12,002 - ERROR - foqus.__main__ - Error importing PyQt
Traceback (most recent call last):
  File "./foqus.py", line 47, in guiImport
    from PyQt5.QtWidgets import QApplication, QSplashScreen, QMessageBox, \
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
...
2019-06-04 22:41:12,488 - ERROR - foqus.__main__ - PyQt5 or Qt not available
2019-06-04 22:41:12,489 - DEBUG - foqus.foqus_lib.framework.sim.turbineConfiguration - Stopped all running consumers
```

When starting foqus.py.  I think this is because I'm on a VM (where I don't have a real video card) so `libgl1-mesa-glx` "fakes" it well enough.

Not sure if this should be documented somewhere?

**Comment by ksbeattie** [comment]
@eslickj will investigate this a little more as will @sotorrio1.  But it seems a little note in the install docs couldn't hurt too much.

**Comment by ksbeattie** [comment]
I will experiment with toggling the 2D & 3D acceleration option for the VMs, just in case that makes a difference.  I will also add a note in the docs about it.

**Comment by ksbeattie** [comment]
I also just solved the following problem (running on a remove Ubuntu machine):

```
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.
```
by installing a new package:

    apt-get install libxkbcommon-x11-0

These might be something for a "troubleshooting" chapter, maybe....

**Comment by ksbeattie** [comment]
Addressed by PR #751
